000-template.md

```diff
@@ -0,0 +1,31 @@
+# <Title>
```

Comment:
I see it as a proposal to have a template for proposals. Because it's not part of the proposal itself I would prefer to remove it from this PR (focused on the Strimzi UI) and opening a different PR for it.

Reply:
Of course - I will remove it from this PR and raise it in another (with the readme change to support it)

Reply:
+1

Reply:
#7 opened for this